index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>modern art</title>
+    <meta charset="utf-8"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+    <script language=”JavaScript” type=”text/javascript” src=”jQuery.js”></script>
+    <script language=”JavaScript” type=”text/javascript” src=”script.js”></script>
+  </head>
+  <body>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
+    <blockquote class="blockquote text-center quote">
+      <p class="mb-0 quote-text">"Every artist was first an amateur"</p>
+      <footer class="blockquote-footer">Ralph Waldo Emerson</footer>
+    </blockquote>    
+    <header class="container fluid">
+      <div class="row">
+        <nav class="col-sm-12">
+          <a class="btn btn-default navigation-button" href="#one" role="button">Biography</a>
+          <a class="btn btn-default navigation-button" href="#two" role="button">Art</a>
+          <a class="btn btn-default navigation-button" href="#three" role="button">Awards</a>
+          <a class="btn btn-default navigation-button" href="#images" role="button">Works</a>
+          <a class="btn btn-default btn-sm" href="https://www.facebook.com/">
+            <i class="fa fa-facebook fa-3x icon"></i>
+          </a>
+          <a class="btn btn-default btn-sm" href="https://twitter.com/?lang=pl">
+            <i class="fa fa-twitter fa-3x icon"></i>
+          </a>
+          <a class="btn btn-default btn-sm" href="https://www.instagram.com/">
+            <i class="fa fa-instagram fa-3x icon"></i>
+          </a>
+        </nav>
+      </div>
+    </header>
+    <div class="container">
+      <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">

```

Comment:
sprawdz prosze czy mozesz zmienic to wszystko `carousel-example-generic` na cokolwiek innego, co bedzie nam cos mowilo. To `carousel-example-generic` to jest id ktore uzyli w przykladzie, dla nas wyglada to slabo.

_______________

We wszystkich miejscach oczywiście

Reply:
zmieniłam